auto_comps.py

```diff
@@ -0,0 +1,224 @@
+            output[name_tft] = (ingame_name, board_size)
+    return output
+
+
+def LoadChampionsAndComps(comp_manager: CompsManager):
+    print('Loading champions and comps...')
+    cached_path = os.path.join(os.path.curdir, 'cached_data')
+    if os.path.isdir(cached_path) == False:
+        os.mkdir(cached_path)
+    cached_file_path = os.path.join(cached_path, 'cached')
+    lol_chess_tftnames_and_price = __LoadLoLChessPrices()
+    set_current = lol_chess_tftnames_and_price[0]
+    if os.path.isfile(cached_file_path + set_current + '.json'):
+        print('Loading from cache file...')
+        with open(cached_file_path + set_current + '.json', 'r') as f:
+            comp_manager.champions = json.loads(f.readline())
+            comp_manager.SetCOMPSLoaded(json.loads(f.readline()))
+    else:
+        print('Loading from web... (It can take up to ~1 min)')
+        response_meta_comps = requests.get(LOLCHESS_META_COMPS_URL).text
+        dragon_info_names_and_boardsize = __LoadCommunityDragon()
+        for each in dragon_info_names_and_boardsize:
+            if each in lol_chess_tftnames_and_price[1]:
+                final_name = dragon_info_names_and_boardsize[each][0]
+                final_price = lol_chess_tftnames_and_price[1][each][0]
+                final_code_name = lol_chess_tftnames_and_price[1][each][1]
+                if each in HARD_OVERRIDE_LIST:
+                    final_code_name = HARD_OVERRIDE_LIST[each]
+                final_boardsize = dragon_info_names_and_boardsize[each][1]
+                comp_manager.champions[final_name] = {'Gold': final_price, 'Board Size': final_boardsize}
+        lol_chess_comps = __LoadLolChessComps(response_meta_comps, set_current, comp_manager)
+        comp_manager.comps_loaded = lol_chess_comps
+        jsoned_champions = json.dumps(comp_manager.champions)
+        jsoned_comps = json.dumps(comp_manager.comps_loaded)
+        with open(cached_file_path + set_current + '.json', 'w') as f:
+            f.write(jsoned_champions + '\n')
+            f.write(jsoned_comps)
+    print('Set: ' + set_current + ', loaded champions: ' + str(len(comp_manager.champions)) + ', comps: ' + str(
+        len(comp_manager.comps_loaded)))
+    for i in range(0, len(comp_manager.comps_loaded)):
+        temp = ','.join(comp_manager.comps_loaded[i][1])
+        print(str(i) + ' - ' + comp_manager.comps_loaded[i][0] + ' [' + temp + ']')
+
+    inputed = ''
+    temp_inputed = ''
+    inputed_file_path = os.path.join(cached_path, 'inputed')
+    if os.path.isfile(inputed_file_path):
+        with open(inputed_file_path, 'r') as f:
+            temp_inputed = f.read()
+        if temp_inputed != '':
+            print(
+                'Your last selection was: "' + temp_inputed + '", press Enter to use last selection, or type "n" and press Enter to make new selection')
+            if input().lower() == 'n':
+                inputed = ''
+            else:
+                inputed = temp_inputed
+
```

Comment:
f-string + if expression instead of statement
```suggestion
                f'Your last selection was: "{temp_inputed}", press Enter to use last selection, or type "n" and press Enter to make new selection'
            )
            inputed = '' if input().lower() == 'n' else temp_inputed
```